src/System.Private.Xml/src/System/Xml/Schema/XmlSchemaComplexType.cs

```diff
@@ -36,11 +36,7 @@ public class XmlSchemaComplexType : XmlSchemaType
 
         static XmlSchemaComplexType()
```

Comment:
I'm surprised this didn't just turn into `private static XmlSchemaComplexType CreateUntypedAnyType()`

Reply:
Good catch.  Will fix. Thanks.